massgov/mayflower
# DP-24276 [a11y] Alart notification

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

- Remove `aria-label` from the header alert and the action step buttons.
- Add `aria-hidden="true"` to `.ma__header-alerts__header__toggle` since this information is causing confusion to screen reader users. As `aria-expanded` tells the state of the accordion, the toggle content is not necessary to be announced.
- Remove the repetitive title on the header alert show/hide toggle.
- Remove the action step toggle "+" symbol from css content, and set it up as HTML to maintain the visual cue to sighted users and to hide from screen readers since it's announced as "plus" and that's delivering confusing information.
- Adjust the right end of the header alert box to line up with other sections.

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-24276)
- [Github issue]()

## Steps to Test


1. Go to /patterns/05-pages-location-park-content/05-pages-location-park-content.html.
2. Open inspect and check:
       - `.ma__header-alerts__header__button` doesn't have `aria-label`, but it has `aria-expanded`.
       - `.ma__header-alerts__header__toggle` has `aria-hidden="true"`.
       - `.ma__action-step__header js-accordion-link` doesn't have `aria-label`, but it has `aria-expanded`.
       - The "+" symbol is in the markup as `.ma__action-step__header__toggle`.  It has `aria-hidden="true"`.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
